### PR TITLE
Take 5 landing page updates

### PR DESCRIPTION
**Pull request by jgagne**
## What this PR does

- Updates Take 5 landing page design; see https://github.com/gymnasium/teamwork/issues/99 for details/tasks
- Updates  Take 5 landing page content; see https://github.com/gymnasium/teamwork/issues/100 for details/tasks

### Content Updates Preview

[Content preview](https://deploy-preview-775--thegymcms.netlify.app/courses/take5/) based on[ design update](https://github.com/gymnasium/teamwork/issues/99).

### Header, Navigation, Featured Section Updates

- Updated categories
- Updated layout (nav left, featured right) 
- Merged Prototyping into UX
- Separated Design and Development 

**Before:**
gym-take5-catalog-header-nav-featured-before

**After:**
gym-take5-catalog-header-nav-featured-after

### Categories Update

- Merged Prototyping into UX
- Separated Design and Development 
- Added category/topic subheading
- Shortened short description
- Added call-to-action: Watch Video 
- Added format
- Added skill level 

**Before:**
gym-take5-category-ux-design-development-before

**After:**
gym-take5-category-ux-after
gym-take5-category-design-deveopment-after

### Short Descriptions, Instructor Bio Updates

- Removed instructor bio language from all short descriptions
- Expanded instructor bio language on details page
- Added category/topic subheading 

**Before:**
gym-take5-details-instructor-description-1-before

**After:**
gym-take5-details-instructor-description-1-after

**Before:**
gym-take5-details-instructor-description-2-before

**After:**
gym-take5-details-instructor-description-2-after

### CODE REVIEW

File: _includes/take5/recommended-content.html [hunk 1]
```diff
@@ -49,26 +49,35 @@
...
-  {%- if item.topic == this_topic -%}
-
-        {%- if item.live == true -%}
-            {%- if item.course_ID != this_tutorial -%}
-              {%- assign recommendations = recommendations | push: item.course_ID -%}
-            {%- endif -%}
-        {%- endif -%}
+{%- if item.topic == this_topic -%}
+  {%- if item.live == true -%}
+      {%- if item.course_ID != this_tutorial -%}
```
Comment (rediris) [inline _includes/take5/recommended-content.html:54]: There's a little inconsistency with the indentation around here...

File: _layouts/take5-raw.html [hunk 2]
```diff
@@ -32,10 +32,11 @@
                 <div class="col-md-8">
                   <div class="take5--details" id="tutorial-details">
                     <header>
+                      <p class="subhead-topic all-caps">{{ course.topic }}</p>
                       <h1 class="title all-caps">{{ course.title }}</h1>
 
                       {% if course.instructor !=null %}
-                      <p class="instructor">with {{ course.instructor }}</p>
+                      <p class="instructor"><b>with {{ course.instructor }}, {{ course.instructor_bio }}</b></p>
```
Comment (rediris) [inline _layouts/take5-raw.html:39]: Should we use strong tags instead of b?

**Review by rediris — APPROVED** [review]
See minor notes. Otherwise, looks good to me!

**Review by andrewpmiller — APPROVED** [review]
Looks good! I want to see it on staging!!!

**Comment by jgagne** [comment]
@rediris I’ll update indentation after the merge. Also, I’d like to stick with `b` over `strong` since it’s more of a cosmetic, typographic display style rather than a semantic reason. We could later remove `b` and add the font weight in the CSS instead.

**Comment by rediris** [comment]
> @rediris I’ll update indentation after the merge. Also, I’d like to stick with `b` over `strong` since it’s more of a cosmetic, typographic display style rather than a semantic reason. We could later remove ‘b’ and add the font weight in the CSS instead.

Totally understood, sounds good!

**Pull request merged**